docfx_project/ziti/clients/linux.md

```diff
@@ -1,59 +1,147 @@
+Restart=always
+RestartSec=3
+
+[Install]
+WantedBy=multi-user.target
+```
+
+
+
+### Enroll Before You Run
+
+You will need the token file or its contents to enroll. Enrollment is the act of exchanging the token for an identity that is subsequently installed permanently in the filesystem.
+
+```bash
+# enroll from a token file
+./ziti-edge-tunnel enroll --jwt ./myTunneler.jwt --identity ./myTunneler.json
+```
+
+```bash
+# enroll from stdin
+./ziti-edge-tunnel enroll --jwt - --identity ./myTunneler.json < ./myTunneler.jwt
+```
+
+### Global Options
 
-`ziti-tunnel` fails to start if no intercept modes can be successfully initialized:
 
-    $ sudo ziti-tunnel run ziti.json
-    [   0.000]    INFO ziti/tunnel/cmd/ziti-tunnel/subcmd.run: tproxy initialization failed: failed to initialize iptables handle: exec: "iptables": executable file not found in $PATH
-    [   0.001]    INFO ziti/tunnel/cmd/ziti-tunnel/subcmd.run: tun initialization failed: failed to open tun interface (name='', mtu=0): ioctl failed with 'invalid argument'
-    [   0.001]   FATAL ziti/tunnel/cmd/ziti-tunnel/subcmd.run: failed to initialize an interceptor
+```bash
+# Load a single identity.
+--identity <identity>
+```
+
+```bash
+# Load all identities in a dir, ignoring files with a .bak or .original filename suffix.
+--identity-dir <dir>
+```
 
-### Intercept Modes
+```bash
+# Set log level, higher is more verbose (default level 3 means INFO).
+--verbose N
+```
+
+```bash
+# Set service polling interval in seconds (default 10).
+--refresh N
+```
+
+### Run Mode
+
+`ziti-edge-tunnel run` provides a transparent proxy and nameserver. The nameserver may be configured to be authoritative (the default) or recursive with a command-line option. The OS is automatically configured to treat the nameserver as primary. You may inspect the resulting configuration with these commands.
+
+```bash
+resolvectl dns     # inspect the association of tun device and nameserver
+resolvectl domain  # inspect the configuration of search domains
+```
+
+The configured tun device will ordinarily have the wildcard search domain configured, depending on the pre-existing resolver configuration and the version of systemd.
+
+```bash
+# Specify the tun interface address and the subnet to which Service domain names are resolved (default 100.64.0.1/10). The nameserver address is always the tun interface address +1, default is 100.64.0.2.
+--dns-ip-range <ip range>
+```
 
-#### tproxy
+#### Run Mode System Requirements
 
-tproxy is the preferred intercept mode when running on a Linux kernel that has the __ip_tables__ kernel module installed.
+`ziti-edge-tunnel run` requires elevated privileges for managing the `/dev/net/tun` device, IP routes, and works best when `libsystemd` is available so that it can call the `systemd` API `/var/run/dbus/system_bus_socket` to configure nameservers and search domains for `systemd-resolved`.

```

Comment:
> and works best when `libsystemd` is available so that it can call the `systemd` API `/var/run/dbus/system_bus_socket`

please remove this. it's in the weeds too far. It's better to state the order of precedence which the tunneler operates. It tries to use the systemd API if it's there, then tries to use `resolvectl`, then it falls back to mucking with the resolve.conf file. And when you do that it's not really a 'system requirement' and probably belongs more in the prior section. 

the only 'requirement' I can see is that you really need elevated privs. we should callout CAP_NET_ADMIN or root - assuming CAP_NET_ADMIN is still enough?

Reply:
Markdown headings should be unique, and so I've used the more specific heading to avoid a collision with the one about Run-Host Mode